src/components/atoms/pageTitle/PageTitle.tsx

```diff
@@ -2,14 +2,31 @@ import { Container, SxProps, Typography } from '@mui/material'
 
 type PageTitleProps = {
   title: string
+  titleUnderlineColor?: string
```

Comment:
Some titles have a colored bar underneath. Add optional prop fore that color. If color is present, render bar